lib/plugins/nextcloud-vue/rules/no-long-imports.ts

```diff
@@ -0,0 +1,78 @@
+/*!
+ * SPDX-FileCopyrightText: 2025 Nextcloud GmbH and Nextcloud contributors
+ * SPDX-License-Identifier: AGPL-3.0-or-later
+ */
+import { Rule } from 'eslint'
+import { createLibVersionValidator } from '../utils/lib-version-parser.js'
+
+// ------------------------------------------------------------------------------
+// Rule Definition
+// ------------------------------------------------------------------------------
+
+/*
+ Introduced in @nextcloud/vue v8.23.0
+ https://github.com/nextcloud-libraries/nextcloud-vue/pull/6385
+ */
+
+const rule: Rule.RuleModule = {
+	meta: {
+		docs: {
+			description: 'Deprecated @nextcloud/vue import syntax',
+			category: 'Nextcloud',
+			recommended: true,
+		},
+		fixable: 'code',
+		schema: [
+			{
+				// We accept one option which is an object
+				type: 'object',
+				properties: {
+					// if we should try to find an appinfo and only handle APIs removed before the max-version
+					parseAppInfo: {
+						type: 'boolean',
+					},
+					// Set a Nextcloud target version, only APIs removed before that versions are checked
+					targetVersion: {
+						type: 'string',
+					},
+				},
+				additionalProperties: false,
+			},
+		],
+		messages: {
+			removedGlobal:
+				'The global property or function {{name}} was removed in Nextcloud {{version}}',
+		},
+	},
+
+	create: function(context) {
```

Comment:
To use ES2015+ syntax

```suggestion
	create(context) {
```